soci/artifacts.go

```diff
@@ -57,6 +57,7 @@ var (
 	bucketKeyPlatform       = []byte("platform")
 	bucketKeyLocation       = []byte("location")
 	bucketKeyType           = []byte("type")
+	bucketKeyORASArtifact   = []byte("oras_artifact")
```

Comment:
Can we store/display the the media type instead? 

If we expect ORAS to be replaced with ORT, then we probably don't want to anchor our code and CLI to ORAS. And, while I don't expect there to be a third media type in this vein, storing this info as a boolean implicitly ties us to exactly 2 options, one of which is ORAS. It's not unfixable, but we don't need to bake in that inflexibility.

Reply:
Fair point. I'll update it.

Reply:
Done.